### DISCUSSION TITLE
PODMAN_COMPOSE_PROVIDER on Windows

### DISCUSSION THREAD

**Discussion by kurtisane** [containers/podman#21215] [Q&A]
I installed Podman Desktop and everything that comes with it. 

I can use podman compose but it defaults to be using docker compose. 
I would like to use podman compose. 

Do I need to seperatly install podman-compose and than set the environment variable ? 
The documentation seems to be old : https://podman-desktop.io/docs/compose/setting-up-compose

**Comment by rhatdan** [comment]
Podman-compose is a different project then Podman, so it would need to be installed separately.
